charts/ad2image/README.md

```
@@ -16,50 +16,40 @@ The command deploys ad2image on the Kubernetes cluster with some default configu
+| podSecurityContext         | object | `{"fsGroup":1000,"runAsGroup":1000,"runAsUser":1000}` | Security Context                                                                                                                       |
+| service.annotations        | object | `{}`                                                  | Service annotations                                                                                                                    |
+| service.port               | int    | `3000`                                                | Service port                                                                                                                           |
+| service.type               | string | `"ClusterIP"`                                         | Service type                                                                                                                           |
+| serviceAccount.annotations | object | `{}`                                                  | Service account annotations                                                                                                            |
+| serviceAccount.create      | bool   | `true`                                                | Create service account                                                                                                                 |
+| serviceAccount.name        | string | `""`                                                  | Service account name                                                                                                                   |
+| defaultMode                | string | `""`                                                  | To overwrite the default mode.                                                                                                         |
+| ad.url                     | string | `""`                                                  | Connection URL for AD server, for example 'ldaps://ad.mydomain.com:636'.                                                               |
+| ad.userSearchBase          | string | `""`                                                  | User Search Base for user lookup, for example 'OU=Users,DC=mycompany,DC=com'.                                                          |
+| ad.userSearchFilter        | string | `"(&(objectClass=organizationalPerson)(cn={uid}))"`   | User Search filter, {uid} will be replaced with the requested user uid.                                                                |
+| ews.ewsServiceUrl          | string | `""`                                                  | EWS service URL, e.g. https://computer.domain.contoso.com/EWS/Exchange.asmx.                                                           |
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)